File: 2020/Python/J3.py
L = int(input())
dropsX,dropsY = [],[]
for counter in range (L):
    paintdrop = input().split(", ")
    dropsX.append(int(paintdrop[0]))
    dropsY.append(int(paintdrop[1]))
print (str(min(dropsX)-1) + ", " + str(min(dropsY)-1))
print (str(max(dropsX)+1) + ", " + str(max(dropsY)+1))

File: 2020/Java/J3.java
import java.util.*;
public class J3 {
	public static void main(String args[]) {
        Scanner input = new Scanner (System.in);
        ArrayList<String> dropsX = new ArrayList<String>();
        ArrayList<String> dropsY = new ArrayList<String>();
		String[] splitStr;
        int L = input.nextInt();
        String str;

        for (int i=0; i<=L;i++){
            str = input.nextLine();
			splitStr = str.split(", ");
			System.out.println(splitStr[0]);
			System.out.println(splitStr[1]);
			/*
            dropsX.add(splitStr[0]);
			dropsY.add(splitStr[1]);
			*/
        }

    }
}
